<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageMonitorForm</class>
 <widget class="QWidget" name="MessageMonitorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>478</width>
    <height>473</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_Main">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_CheckBoxes">
       <property name="leftMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QCheckBox" name="checkBox_SuspendOutput">
         <property name="text">
          <string>Suspend output</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_PagedScroll">
         <property name="text">
          <string>Paged scroll (uglier, lighter to handle, less CPU/GPU-usage)</string>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_NMEA">
         <property name="text">
          <string>NMEA</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_UBX">
         <property name="text">
          <string>UBX</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_RTCM">
         <property name="text">
          <string>RTCM</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_UBXParseErrors">
         <property name="text">
          <string>UBX parse errors</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_NMEAParseErrors">
         <property name="text">
          <string>NMEA parse errors</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_UnidentifiedData">
         <property name="text">
          <string>Unidentified data</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_CenterControls">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_MaxLines">
         <item>
          <widget class="QLabel" name="label_MaxLines">
           <property name="text">
            <string>Max lines:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="spinBox_MaxLines">
           <property name="minimum">
            <number>10</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>1000</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_ClearAll">
         <property name="text">
          <string>Clear all</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit_Output"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
